New Entry: General MFA (#1007)
Codecademy/docs: content/general/concepts/multi-factor-authentication/multi-factor-authentication.md

---
Title: 'Multi-factor Authentication'
Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
Subjects:
  - 'Computer Science'
  - 'Cybersecurity'
Tags:
  - 'Best Practices'
  - 'Cybersecurity'
CatalogContent:

>   - 'paths/code-foundations'

Reviewer: ```suggestion
  - 'introduction-to-cybersecurity'
```

I would add at least one free course/path if possible. This one seemed like the most relevant.

  - 'paths/computer-science'
---

**Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.

## Factors

> One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require user's to create when initially signing up for an account.

Reviewer 2: ```suggestion
One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
```

Other factors, depending on the verification system, include the following:

- A physical item belonging to the user (e.g., a mobile phone).

> - A password, personal identity number (PIN), key, or token that only the user should know.

Reviewer 2: You already mentioned "password." Why not mention authentication codes that are e-mailed or texted or phoned in to the user.

> - A password, personal identity number (PIN), key, or token that only the user should know.

Author: ```suggestion
- A personal identity number (PIN), key, or token that only the user should know prior, or an authentication code that is e-mailed, texted, or phoned-in to the user in real-time.
```
